src/orga/funding/sdg-2023-1.rst

```diff
@@ -0,0 +1,99 @@
+:orphan:
+
+NumFOCUS SDG 2023 Round 1 Proposal
```

Comment:
Maybe we should link to NF's pages to avoiding the clutter here. I'm not against having them here but it will be a chore as the SDGs add up.